os/fs/procfs/fs_procfsproc.c

```diff
@@ -405,6 +411,60 @@ static ssize_t proc_entry_stat(FAR struct proc_file_s *procfile, FAR struct tcb_
 	copysize = procfs_memcpy(procfile->line, linesize, buffer, buflen, &offset);
 	totalsize += copysize;
 
+#ifdef CONFIG_SCHED_CPULOAD
+	for (cpuload_idx = 0; cpuload_idx < SCHED_NCPULOAD; cpuload_idx++) {
+		buffer += copysize;
+		remaining -= copysize;
+		if (totalsize >= buflen) {
+			return totalsize;
+		}
+
+		/* Get load measurement data */
+		(void)clock_cpuload(procfile->pid, cpuload_idx, &cpuload);
+
+		if (cpuload.total > 0) {
+			uint32_t tmp;
+
+			tmp = (1000 * cpuload.active) / cpuload.total;
+			intpart = tmp / 10;
+			fracpart = tmp - 10 * intpart;
+		} else {
+			intpart = 0;
+			fracpart = 0;
+		}
+		linesize = snprintf(procfile->line, STATUS_LINELEN, " %d.%d", intpart, fracpart);
+		copysize = procfs_memcpy(procfile->line, linesize, buffer, remaining, &offset);
+		totalsize += copysize;
+	}
+
+#ifdef CONFIG_SCHED_CPULOAD_SNAPSHOT
+	buffer += copysize;
```

Comment:
There are almost duplicated code from line 416~line 437. Could you make this for one?

Reply:
I will consider it later.